zebrad/tests/acceptance.rs

```diff
@@ -461,3 +461,80 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn metrics_tracing_listeners() -> Result<()> {
+    use hyper::{Body, Client, Request, Uri};
+
+    zebra_test::init();
+
+    // From here we dont know what port the OS selected for us if we use port 0 so we use a hard coded port for the 2 endpoints.
+    let metrics_endpoint = "127.0.0.1:9998";
+    let metrics_url = "http://127.0.0.1:9998";
+    let tracing_endpoint = "127.0.0.1:9999";
+    let tracing_url = "http://127.0.0.1:9999/filter";
```

Comment:
Consider choosing ports from 50000-60000, because applications usually don't bind to those port ranges:
https://en.m.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

Also, let's add a note about how to fix port conflicts.
There is already a comment about port conflicts in another test:
https://github.com/ZcashFoundation/zebra/blob/main/zebrad/tests/acceptance.rs#L447

Let's move that note to the module comment at the top of the file. Then we can refer to it from all 3 tests.

Reply:
https://github.com/ZcashFoundation/zebra/pull/1000/commits/e301ab938643fe8369752e9228fda5094699efbb